### PR TITLE
Feature mealplans page

### PR DESCRIPTION
**Pull request by bambolelooo**

### CODE REVIEW

File: assets/js/services/mealplans_api.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+            $(".btn-tuesday").click(() => addRecipesToDiv(1));
+            $(".btn-wednesday").click(() => addRecipesToDiv(2));
+            $(".btn-thursday").click(() => addRecipesToDiv(3));
+            $(".btn-friday").click(() => addRecipesToDiv(4));
+            $(".btn-saturday").click(() => addRecipesToDiv(5));
+            $(".btn-sunday").click(() => addRecipesToDiv(6));
+        });
+}
+function addRecipesToDiv(day) {
+    console.log(`started adding divs`);
```
Comment (lavanyavirushan) [inline assets/js/services/mealplans_api.js]: Please make sure to remove all console logs for cleaner code

File: assets/js/services/mealplans_api.js [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+            $(".btn-monday").click(() => addRecipesToDiv(0));
+            $(".btn-tuesday").click(() => addRecipesToDiv(1));
+            $(".btn-wednesday").click(() => addRecipesToDiv(2));
+            $(".btn-thursday").click(() => addRecipesToDiv(3));
+            $(".btn-friday").click(() => addRecipesToDiv(4));
+            $(".btn-saturday").click(() => addRecipesToDiv(5));
+            $(".btn-sunday").click(() => addRecipesToDiv(6));
+        });
+}
+function addRecipesToDiv(day) {
```
Comment (lavanyavirushan) [inline assets/js/services/mealplans_api.js]: Please make comments for all functions

File: assets/js/services/mealplans_api.js [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+            let p2 = await p1.json();
+            mealsJSON[meal] = p2;
+        } catch (error) {
+            console.log(error.message);
+            mealsJSON[meal] = "fetch error";
+        }
+    }
+    return mealsJSON;
+}
+// console.log("CALLING FETCHURLS");
```
Comment (lavanyavirushan) [inline assets/js/services/mealplans_api.js]: Please remove all unwanted comments

**Review by lavanyavirushan — CHANGES_REQUESTED** [review]
lets remove all unused files such as:
./assets/js/ui/mealplans.js
./assets/js/dispatcher/mealplans.js

lets remove all console logs
add comments for functions like what it does
removes unwanted comments 

GREAT WORK!!!

**Review by DexterDick — APPROVED** [review]
Good work.

File: assets/html/diet.html [hunk 4]
```diff
@@ -4,113 +4,381 @@
...
+                                <dl
+                                    class="mt-2 flex flex-col md:flex md:flex-wrap text-sm md:text-md font-medium"
+                                >
+                                    <div
+                                        class="meal-type px-1.5 ring-1 ring-slate-200 rounded ring-emerald-200 flex justify-center items-center w-24 m-2"
+                                    >
+                                        Lunch
+                                    </div>
+
+                                    <p class="lunch-ingridients text-slate-200">
```
Comment (lavanyavirushan) [inline assets/html/diet.html:168]: As these values are hard coded. It leaves a bug in the meal plan page. If users simply click on meal plans without doing calorie count these hard coded values are repeating with same data for breakfast, snack1, lunch, snack2, dinner. this either have to be empty value and asking users to do self evaluation first or by default show some random meal plan. I'll leave that to you.

JS looks good.

**Review by lavanyavirushan — CHANGES_REQUESTED** [review]
As meals in the HTML are hard coded values initially. It leaves a bug in the meal plan page. If users simply click on meal plans without doing self evaluation, these hard coded values are repeating with same data for breakfast, snack1, lunch, snack2, dinner. this either have to be empty value and asking users to do self evaluation first or by default show some random meal plan. I'll leave that to you.

**Review by nnavg — APPROVED** [review]
looks good

**Review by DexterDick — APPROVED** [review]
looks good

**Review by lavanyavirushan — APPROVED** [review]
looks good

**Review by DexterDick — COMMENTED** [review]
looks good

**Pull request merged**